TensorMath.lua

```diff
@@ -388,6 +388,18 @@ wrap("add",
         {name=real, default=1},
         {name=Tensor}})
 
+
+wrap("csub",
+     cname("sub"),
```

Comment:
I think it's okay. the sub is the cname, not the lua name (which is csub). We have it the same in torch7 https://github.com/torch/torch7/blob/master/TensorMath.lua#L259-L268